PluginDirectories/1/fpm.bundle/info.json

```diff
@@ -0,0 +1,14 @@
+{
+  "categories" : [
+    "Utilities",
+    "Package Management",
```

Comment:
That is really up to you. Maybe you could support the concept of tags along with categories and somehow enforce the restriction on what categories one can use in the categories array. I don't know. Having a model where any developer is free to make up their own categories but you would like to maintain a fixed list of categories is kind of hard to maintain. You'd have to basically add this very same comment to any pull request for plugins.